specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/workspaceRP.json

```diff
@@ -3409,6 +3409,14 @@
             "read"
           ]
         },
+        "serverlessComputeCustomSubnet": {
```

Comment:
can you rename this property and format it as an arm-id?

[https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details)

Reply:
Keeping the name and updated the format as arm-id